builtin/logical/aws/path_config_root.go

```diff
@@ -50,6 +51,35 @@ func pathConfigRoot(b *backend) *framework.Path {
 	}
 }
 
+func (b *backend) pathConfigRootRead(ctx context.Context, req *logical.Request, data *framework.FieldData) (*logical.Response, error) {
+	b.clientMutex.RLock()
+	defer b.clientMutex.RUnlock()
+
+	entry, err := req.Storage.Get(ctx, "config/root")
+	if err != nil {
+		return nil, err
+	}
+	var config rootConfig
+
+	if entry != nil {
+		if err := entry.DecodeJSON(&config); err != nil {
+			return nil, err
+		}
+	}
```

Comment:
If `entry == nil` I think we should return `nil, nil`.

Reply:
done